app/i18n/locales/en.json

```diff
@@ -734,6 +737,10 @@
   "Cannot_leave": "Cannot leave",
   "Last_owner_team_room": "You are the last owner of this channel. Once you leave the team, the channel will be kept inside the team but you will be managing it from outside.",
   "last-owner-can-not-be-removed": "Last owner cannot be removed",
+  "Removing_user_from_this_Team": "You are removing {{user}} from this Team",
```

Comment:
Try not to uppercase words unnecessary.
```
You are removing {{user}} from this team
```

It's ok to capitalize every word on a list item (or title like `Remove Member` below), but not on this case.